### PR TITLE
[LEA-2765] Extension is sending x-api-version

### PR DESCRIPTION
**Pull request by apiotrowski312**

### CODE REVIEW

File: zap/middleware.go [hunk 1]
```diff
@@ -45,7 +45,15 @@ func (w *Logger) EchoMiddleware(l logi.WLogger) echo.MiddlewareFunc {
 				key.URI, req.RequestURI,
 				key.Status, res.Status,
 				key.UserAgent, req.UserAgent(),
-				key.APIVersion, req.Header.Get("X-API-Version"),
+				key.APIVersion, res.Header().Get("API-Version"),
```
Comment (sonereker) [inline zap/middleware.go:48]: Why did you remove `X-` it's the convention to name non-standard headers?

Comment (sonereker) [reply zap/middleware.go:48]: Hmm https://tonyxu.io/posts/2018/http-deprecate-x-prefix/

**Review by sonereker — APPROVED** [review]

**Review by rginestou — APPROVED** [review]
